taichi/transforms/simplify.cpp

```diff
@@ -71,57 +71,9 @@ class BasicBlockSimplify : public IRVisitor {
       }
     }
 
-    // weaken indexing
```

Comment:
This part simplifies `LoopIndex + constant(a) + constant(b) + constant(c)` into `LoopIndex + constant(a + b + c)`.